cs251tk/toolkit/args.py

```diff
@@ -166,6 +165,15 @@ def process_args():
     parser = build_argparser()
     args = vars(parser.parse_args())
 
+    if args['web']:
+        args['highlight_partials'] = True
+        if not len(args['to_record']) == 1:
```

Comment:
```python
if not args['to_record'] or len(args['to_record']) != 1 or len(args['to_record'][0]) != 1:
```

maybe? Otherwise we'll still need to add the first check, to prevent the `[0]` from throwing, and I'd still like to switch these to `!= 1` instead of `not … == 1`.